### PR TITLE
Load theme on Blue Ocean

### PR DESCRIPTION
**Pull request by timja**
Part of https://github.com/jenkinsci/dark-theme-plugin/issues/89

### CODE REVIEW

**Comment by timja** [comment]
> Should the theme extension have an option to decide whether it wants to be enabled for BlueOcean or not?

I was wondering about that, sure

**Comment by oleg-nenashev** [comment]
I would recommend disabling themes for Blue Ocean by default, because it
has a different CSS structure which still needs an update IMHO.

File: src/main/java/io/jenkins/plugins/thememanager/Theme.java [hunk 1]
```diff
@@ -79,6 +88,7 @@ public static Builder builder() {
   public static class Builder {
     private List<String> cssUrls = emptyList();
     private List<String> javascriptUrls = emptyList();
+    private boolean blueOceanCompatible = true;
```
Comment (oleg-nenashev) [inline src/main/java/io/jenkins/plugins/thememanager/Theme.java]: No strong opinion, but I would rather prefer opt-in at the current stage

```suggestion
    private boolean blueOceanCompatible = false;
```

Comment (timja) [reply src/main/java/io/jenkins/plugins/thememanager/Theme.java]: already changed, saw your comment after i'd pushed

**Comment by timja** [comment]
and it works 😂

**Comment by oleg-nenashev** [comment]
> and it works 😂

I won't believe until I see the screenshots :P

**Comment by timja** [comment]
😂 

image

**Comment by timja** [comment]
BlueOcean development is painful, haven't managed to get it live reloading.

Only way I've managed to get css changes to work is to update is to run:

```
mvn clean install -D skipTests -Dspotbugs.skip
```

on the `jenkins-design-language` submodule which takes ~50 seconds -.-

**Comment by oleg-nenashev** [comment]
> BlueOcean development is painful, haven't managed to get it live reloading.

https://chrome.google.com/webstore/detail/live-editor-for-css-less/ifhikkcafabcgolfjegfcgloomalapol?hl=en and so on?

**Comment by timja** [comment]
> > BlueOcean development is painful, haven't managed to get it live reloading.
> 
> https://chrome.google.com/webstore/detail/live-editor-for-css-less/ifhikkcafabcgolfjegfcgloomalapol?hl=en and so on?

probably not the most helpful as it's less files that need to be changed, 

but maybe for prototyping.

It's looking like this now anyway:

image

**Pull request merged**